nilearn/decoding/tests/test_decoder.py

```diff
@@ -257,33 +256,38 @@ def test_decoder_classification_string_label():
 
 
 def test_decoder_regression():
-    X, y = make_regression(n_samples=200, n_features=125,
-                           n_informative=5, noise=0.2, random_state=42)
+    dim = 30
+    X, y = make_regression(n_samples=100, n_features=dim**3, n_informative=dim,
+                           noise=1.5, bias=1.0, random_state=42)
     X = StandardScaler().fit_transform(X)
-    y = (y - y.mean()) / y.std()
-    X, mask = to_niimgs(X, [5, 5, 5])
-    for regressor_ in regressors:
-        for screening_percentile in [100, 20]:
-            model = DecoderRegressor(estimator=regressor_, mask=mask,
+    X, mask = to_niimgs(X, [dim, dim, dim])
+    for reg in regressors:
+        for screening_percentile in [100, 20, 1]:
+            model = DecoderRegressor(estimator=reg, mask=mask,
                                      screening_percentile=screening_percentile)
             model.fit(X, y)
             y_pred = model.predict(X)
             assert r2_score(y, y_pred) > 0.95
-        for clustering_percentile in [100, 99]:
-            screening_percentile = 90
-            model = fREMRegressor(estimator=regressor_, mask=mask,
-                                  clustering_percentile=clustering_percentile,
-                                  screening_percentile=screening_percentile,
-                                  cv=10)
-            model.fit(X, y)
-            y_pred = model.predict(X)
-            assert r2_score(y, y_pred) > 0.95
+
+    dim = 5
```

Comment:
Should we keep tests with both `dim = 5` and `dim = 30` ? Aren't they redundant ?

Reply:
Yes this can be done; I might just want for the `clustering_percentile` change to speed up a bit so I changed back to dim = 5;

Keeping `dim > 30` is much better to replicate settings real dataset but test cases will be slower to run.

Reply:
I'm not sure that `dim=30` is more useful than `dim=5` so if it's time-consuming, let's get rid of it.

Reply:
This is my purpose when using 2 `dim` case: first is to replicate failure on current master branch on using `screening_percentile` parameter`; second one is `dim=5` is the similar option with other test cases.

Reply:
Ah, you cannot reproduce the problem with 'dim' less than 30 ?
I'm just worried about the cost.

Reply:
Yes only upping `dim=30` and `screening_percentile < 15` will reproduce the problem with `master` branch.

Reply:
OK, let's leave it here then.